source/frontend/src/features/mapSideBar/acquisition/common/AcquisitionMenu.tsx

```diff
@@ -43,17 +44,20 @@ const AcquisitionMenu: React.FunctionComponent<
     <>
       <StyledMenuWrapper>
         {props.items.map((label: string, index: number) => {
+          const activeIndex = props.selectedIndex === index;
           if (index === 0) {
             return (
               <StyledRow
                 key={`menu-item-row-${index}`}
                 data-testid={`menu-item-row-${index}`}
                 className={cx('no-gutters', { selected: props.selectedIndex === index })}
               >
-                <Col xs="1">{props.selectedIndex === index && <FaCaretRight />}</Col>
-                <Col onClick={() => (props.selectedIndex !== index ? handleClick(index) : '')}>
-                  {label}
-                </Col>
+                {activeIndex && <Col>{label}</Col>}
```

Comment:
This would actually be a good use of a ternary, to indicate that only one of two options can be displayed at a time.

Reply:
Updated